_posts/2022-10-06-audio-classification-torch/audio-classification.Rmd

```diff
@@ -0,0 +1,472 @@
+---
+title: "Audio classification with torch"
+description: >
+  Learn how to classify speech utterances with torch, making use of domain knowledge and deep learning. This post is a condensed version of the same-topic chapter in the book to be published with CRC Press, Deep Learning and Scientific Computing with R torch. 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatorchaudio
+date: 2022-10-06
+categories:
+  - Torch
+  - R
+  - Audio Processing
+bibliography: bibliography.bib
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/squirrel.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE, fig.width = 6, fig.height = 6)
+```
+
+## Variations on a theme
+
+[Simple Audio Classification with Keras](https://blogs.rstudio.com/ai/posts/2018-06-06-simple-audio-classification-keras/), [Audio classification with Keras: Looking closer at the non-deep learning parts](https://blogs.rstudio.com/ai/posts/2019-02-07-audio-background/), [Simple audio classification with torch](https://blogs.rstudio.com/ai/posts/2021-02-04-simple-audio-classification-with-torch/): No, this is not the first post, on this blog, that introduces speech classification using deep learning. With two of those posts (the "applied" ones) it shares the general setup, the type of deep-learning architecture employed, and the dataset used. With the third, it has in common the interest in the ideas and concepts involved. Each of these posts has a different focus, -- should you read this one?
```

Comment:
It looks like the three of you actually do use different capitalization rules for your post titles. But, since you and Athos are in the majority, I think Daniel can get overriden here for consistency's sake.

Reply:
:-))